hack/make/.platforms

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# List of platforms to cross compile
+export DOCKER_CROSSPLATFORMS="
+    linux/386 linux/arm
+	darwin/amd64
+	freebsd/amd64 freebsd/386 freebsd/arm
+	windows/amd64 windows/386"
```

Comment:
Indentation here is a little odd (mixed tabs/spaces) -- also, I think this might need to be conditionally set based on whether it's set from the environment (`docker run -e DOCKER_CROSSPLATFORMS=... docker-dev...` is supported currently), ie:
```bash
: ${DOCKER_CROSSPLATFORMS:="
	linux/386 linux/arm
	darwin/amd64
	freebsd/amd64 freebsd/386 freebsd/arm
	windows/amd64 windows/386
"}
export DOCKER_CROSSPLATFORMS
```